docs/shared/recipes/plugins/project-graph-plugins.md

```diff
@@ -80,6 +80,57 @@ If you create targets for a project within a plugin's code, the Nx migration gen
 
 {% /callout %}
 
+#### Example (extending projects / adding targets)
```

Comment:
Why would plugin authors want to only extend a project and not create a new project if it doesn't exist? I think it is probably a valid case, but you should have a compelling answer to that question in this intro text.